src/components/SingleReview.jsx

```diff
@@ -18,6 +20,33 @@ export default function SingleReview({id}) {
     const date = new Date(review.created_at)
     const postedAt = (date.toLocaleString());
 
+    function handleVoteClick(e) {
+        e.preventDefault()
+        const change = Number(e.target.value)
+        setReview((review) => {
+            const reviewCopy = {...review}
+            reviewCopy.votes = review.votes + change
+            reviewCopy.comments = review.comment_count
+            return reviewCopy
+        })
+        api.patchReviewVotes(change,id)
+        .then(response => setReview(review => {
```

Comment:
We don't want to be setting our review state after the patch request here. If multiple other users have voted at the same time, we may get a result back that has an unexpected vote increase. Instead, we can just handle this optimistically as you've done before this